NAMESPACE

```diff
@@ -72,6 +72,7 @@ export(run_gtest)
 export(set_log_throw_on_error)
 export(setup_and_run_gtest)
 export(setup_gtest)
+export(transform_steepness)
```

Comment:
add `#' @export transform_steepness` to FIMS-package.R so this export is saved if NAMESPACE is re-written.

Reply:
transform_steepness is an R function not a C++ function so I use @export in the roxygen to generate line 75.

Reply:
perfect!

Reply:
Your comment is now super relevant and helped shape the outcome of this PR so thank you 🙏.